app/views/rules/_form.html.erb

```diff
@@ -132,7 +132,13 @@
   });
   secondaryButtons.push(cancelButton);
 
-  <% unless @rule.persisted? %>
+  <% if @rule.persisted? %>
+    var activityButton = Button.create(app, { label: 'Activity' });
+    activityButton.subscribe(Button.Action.CLICK, () => {
+      Turbolinks.visit('<%= activity_rule_path(@rule) %>');
+    });
+    secondaryButtons.push(activityButton);
+  <% else %>
```

Comment:
As discussed, I'm happy to revisit the links in a future iteration. Give it a go, if you want to.